### ISSUE TITLE
Please Fix Forester's Main Mechanic

### ISSUE THREAD

**Issue by mccabefliam** [Vek17/WrathMods-TabletopTweaks#399]
**Is your feature request related to a problem? Please describe.**
Forester's main feature, animal focus, doesn't apply the passive enhancement bonus. It only lets you get it as an active basically defeating the whole point of the archtype. 

**Describe the solution you'd like**
Forester getting a permanent enhancement bonus of their choice (since the companion is always dead) and the ability to further buff themselves with the active (with a different bonus like bull or bear while the main one is cat). 

**Describe alternatives you've considered**
I considered just using toybox to slap on the bonus but its tedius that it doesn't work like intended and that I can't switch bonuses on the fly. 

**Additional context**
The selection comes up and claims to be applying but stats don't change and nothing is affected.

**Comment by usrmne** [comment]
Honestly, would he be viable even with that fixed?

**Comment by mccabefliam** [comment]
Not about playing on the hardest difficulties. If you make the easier difficulties harder with worse classes it levels out.

**Comment by decourcy2** [comment]
Yes, this is an important fix, and yes, the forester is a great arch.

**Comment by Bezhukov** [comment]
> Honestly, would he be viable even with that fixed?

If Favored Terrain progression gets fixed too, sure. Tactician is pretty good in a high rest environment and it gets every ability under the sun plus Bonus Feats. Hunter Class as a whole would be helped immensely if Ranged Teamwork Feats were implemented but Forester can also go Lead Blades two-handed melee.
